src/components/About/About.js

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { Grid, makeStyles, ButtonBase, Link } from '@material-ui/core';
+import theme from '../../theme/';
+import { useTranslation } from 'react-i18next';
+import { SUPPORT_URL, COSMOTECH_URL } from '../../config/AppConfiguration';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    alignItems: 'center',
+  },
+  picture: {
+    marginRight: '16px',
+  },
+  title: {
+    fontWeight: 'bold',
+    fontSize: '35px',
+    marginBottom: '16px',
+  },
+  version: {
+    fontWeight: 'bold',
+  },
+  content: {
+    marginTop: '16px',
+    marginBottom: '16px',
+  },
+}));
+
+export const About = () => {
+  const { t } = useTranslation();
+  const classes = useStyles();
+
+  return (
+    <Grid container spacing={2} className={classes.root}>
+      <Grid item className={classes.picture}>
+        <ButtonBase>
+          <img alt="Cosmo Tech" src={theme.picture.logo} />
+        </ButtonBase>
+      </Grid>
+      <Grid item container xs>
+        <Grid item container direction="column">
+          <Grid className={classes.title} item spacing={2}>
+            {t('genericcomponent.dialog.about.title')}
+          </Grid>
+          <Grid item className={classes.version}>
+            2.2.0-brewery
```

Comment:
We should not use a hard-coded string to set the application version, otherwise all solutions will have conflicts every time this value is updated.
Could you try to read this value from _package.json_ ? (c.f. [this SO post](https://stackoverflow.com/questions/45978230/get-version-number-from-package-json-in-react-redux-create-react-app?answertab=votes#tab-top) for some leads)